phantom.coffee

```diff
@@ -61,7 +61,7 @@ module.exports =
     if typeof options.parameters is 'object'
       for key, value of options.parameters
         args.push '--'+key+'='+value
-    options.path ?= ''
+    options.path ?= './node_modules/phantomjs/bin/'
```

Comment:
You should be able to use `phantomjs.path` all the time. I did some tests and `phantomjs` does fall back to the system binary if it exists.